### PR TITLE
Silence warnings about unused variables

### PR DESCRIPTION
**Pull request by pash7ka**
Fixes compiler warnings abount unused local variables. Issue #367

### CODE REVIEW

File: contracts/ownership/HasNoTokens.sol [hunk 1]
```diff
@@ -13,11 +13,11 @@ contract HasNoTokens is CanReclaimToken {
 
  /**
   * @dev Reject all ERC23 compatible tokens
-  * @param from_ address The address that is transferring the tokens
-  * @param value_ uint256 the amount of the specified token
-  * @param data_ Bytes The data passed from the caller.
+  * param from_ address The address that is transferring the tokens
```
Comment (frangio) [inline contracts/ownership/HasNoTokens.sol]: Was there a reason for removing the `@` symbols? Does Solidity not compile with them?

**Comment by pash7ka** [comment]
Yes, with `@` compiler fails 
```
DocstringParsingError: Documented parameter "from_" not found in the parameter list of the function.
,DocstringParsingError: Documented parameter "value_" not found in the parameter list of the function.
,DocstringParsingError: Documented parameter "data_" not found in the parameter list of the function.
Compiliation failed. See above.
```

**Comment by frangio** [comment]
That is very annoying. I'm not sure I prefer this. I'm going to see if something can be changed in Solidity regarding this problem.

**Comment by pash7ka** [comment]
I've also tried another solution:
```
if(true || from_ == 0 || value_ == 0 || data_.length == 0 ) revert(); // Temporary fix for solidity compiler warning
```
But it affects generated code and I really don't like such side effect.

**Comment by frangio** [comment]
I agree. I also thought of something like `assert(from_ == from_);` as a sort of "nop". It should be removed by the optimizer but I haven't verified it.

**Comment by frangio** [comment]
Closing as the solution removing the NatSpec documentation for the variable is not acceptable. Continuing discussion in #367.

**Pull request closed without merging**